build.xml

```diff
@@ -102,6 +102,7 @@
   <target name="spldoctoolkits">
      <exec executable="${streams.install}/bin/spl-make-doc">
         <arg value="--include-composite-operator-diagram"/>
+        <arg value="--include-all"/>
```

Comment:
If I remember correctly, the first time I built the documentation, some of the text for the sample applications was missing, so I added '--include-all'. But that might be overkill.